CedarJava/src/main/java/com/cedarpolicy/serializer/JsonEUID.java

```diff
@@ -30,8 +36,14 @@ public class JsonEUID {
     public final String id;
```

Comment:
should this be `EntityIdentifier`?

Reply:
Don't need to change in this PR, but I think it might now be possible to remove this class? If not, I think it would make sense to store the EntityUID instead of the `type` and `id` separately.

Reply:
I think so as well but wanted to leave that for a larger refactoring